collectors/python.d.plugin/README.md

```diff
@@ -13,7 +13,13 @@
 
 Every module should be compatible with python2 and python3.
 All third party libraries should be installed system-wide or in `python_modules` directory.
-Module configurations are written in YAML and **pyYAML is required**.
+
+**Netdata uses the default Python system installation**, thus for *most* systems (e.g Ubuntu) Python2 will be used.
```

Comment:
Python 2 has a deadline next January 1st, so I think it is already possible to say that Python 3 will be used.

Reply:
I installed netdata in multiple setups and python3 always lead to problems. I remember reading several issues here with the same thing. Maybe for the time being and until python is linked to python3 by various distros, it's better to focus on python2 (UX wise)?

Reply:
All right, let us keep for while, thanks!